engine/runtime/src/main/java/org/enso/compiler/TreeToIr.java

```diff
@@ -276,71 +202,16 @@ case List(expr) => Some(expr)
         var binding = new IR$Module$Scope$Definition$Method$Binding(
           reference,
           nil(),
-          translateExpression(a.getExpr(), false),
+          translateExpression(a.getExpr()),
           getIdentifiedLocation(a),
           meta(), diag()
         );
         yield cons(binding, appendTo);
       }
-    /*
```

Comment:
I was keeping these _scala code comments_ in order to know what the old parse did. Removing them is a signal we no longer need to study the old parser. That is great.